### PR TITLE
mono-zeroconf: remove broken package

### PR DESCRIPTION
**Pull request by delroth**
###### Motivation for this change

No dependencies within nixpkgs, and the package has not built successfully since 2018-04-29 according to Hydra[1]. Last upstream release was in 2009, and last upstream git commit in 2013.

[1] https://hydra.nixos.org/build/100604053

@NixOS/backports if you're interested for #68361

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by worldofpeace — CHANGES_REQUESTED** [review]
Add an alias in `aliases.nix`.
```
mono-zeroconf = throw "deprecated 2019-09-20: abandoned by upstream."
```

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -217,6 +217,7 @@ mapAliases ({
   mlt-qt5 = libsForQt5.mlt;  # added 2015-12-19
   mobile_broadband_provider_info = mobile-broadband-provider-info; # added 2018-02-25
   module_init_tools = kmod; # added 2016-04-22
+  mono-zeroconf = throw "deprecated 2019-09-20: abandoned by upstream."
```
Comment (worldofpeace) [inline pkgs/top-level/aliases.nix]:
```suggestion
  mono-zeroconf = throw "deprecated 2019-09-20: abandoned by upstream.";
```

Comment (delroth) [reply pkgs/top-level/aliases.nix]: I wish git commit --amend would tell me when I've forgotten to git add the change :/ sorry.

**Comment by worldofpeace** [comment]
backported in ea623c7ef8ccc43b642cb7b81a5ce3379719c888

**Pull request merged**